src/launch/linux.rs

```diff
@@ -0,0 +1,3 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod ioctl;
```

Comment:
`launch/linux.rs` ?? are there any other OSes, which can do a launch?

Reply:
The naming structure here is for consistency with how the `/dev/sev` ioctl support is implemented in the crate.

Reply:
Hmm, my personal preference is subdir with `mod.rs`, which would be consistent with the `firmware` subdir.

Also would the `mod linux;` then need a `#[cfg(target_os = "linux")]`?

Reply:
I can add it in the next revision :+1: